.circleci/documentation-versions.py

```diff
@@ -19,15 +46,24 @@ def main():
     except IndexError:
         sys.exit("Missing build dir path: python /path/docs")
 
-    ignored_dirs = {".git"}
+    # directories which contain docs
     dirs = [
         item
         for item in os.scandir(build_dir)
-        if item.is_dir() and item.name not in ignored_dirs
+        if item.is_dir() and VERSION_RE.match(item.name)
     ]
+
     sorted_dirs = sorted(dirs, reverse=True, key=attrgetter("name"))
```

Comment:
If you use `sorted()` on dirs list comp on line 50 you can get rid of this line.